### PR TITLE
Something1

### PR DESCRIPTION
**Pull request by meaghanlewis**

### CODE REVIEW

File: ConsoleApp1/ConsoleApp1/ConsoleApp1.vbproj [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
+  </PropertyGroup>
```
Comment (meaghanlewis) [inline ConsoleApp1/ConsoleApp1/ConsoleApp1.vbproj:6]: Some comment here

File: ConsoleApp1/ConsoleApp1.sln [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+		{F01CC5EF-1808-46FE-85B7-322EC98A2482}.Release|Any CPU.ActiveCfg = Release|Any CPU
+		{F01CC5EF-1808-46FE-85B7-322EC98A2482}.Release|Any CPU.Build.0 = Release|Any CPU
+	EndGlobalSection
+	GlobalSection(SolutionProperties) = preSolution
+		HideSolutionNode = FALSE
+	EndGlobalSection
+	GlobalSection(ExtensibilityGlobals) = postSolution
+		SolutionGuid = {B96B1002-C9FD-4250-B6C6-C53A549E5E79}
+	EndGlobalSection
+EndGlobal
```
Comment (meaghanlewis) [inline ConsoleApp1/ConsoleApp1.sln:25]: Another comment here

**Review by meaghanlewis-tester — APPROVED** [review]

File: TestWebAppProject/TestWebAppProject.csproj [hunk 3]
```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk.Web">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp1.1</TargetFramework>
+    <TargetFramework>netcoreapp1.1.1</TargetFramework>
```
Comment (meaghanlewis) [inline TestWebAppProject/TestWebAppProject.csproj:4]: comment here

Comment (meaghanlewis) [reply TestWebAppProject/TestWebAppProject.csproj:4]: 👍 :thumbsup:

**Review by ghfvs-tester — APPROVED** [review]

**Pull request closed without merging**